app/src/Manager/AdvisorManager.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Manager;
+
+use App\Entity\Game;
+use App\Repository\GameRepository;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Unirest\Request as Request;
+
+class AdvisorManager
+{
+    private const ADVISOR_URL = 'http://advisor:8001';
```

Comment:
It should be in configuration (possibly with the port from an environment variable)